pipettes/firmware/motor_hardware.c

```diff
@@ -29,6 +29,8 @@ void HAL_SPI_MspInit(SPI_HandleTypeDef* hspi) {
         GPIO_InitStruct.Pull = GPIO_NOPULL;
         GPIO_InitStruct.Speed = GPIO_SPEED_FREQ_LOW;
         HAL_GPIO_Init(GPIOC, &GPIO_InitStruct);
+
+        HAL_GPIO_WritePin(GPIOC, GPIO_InitStruct.Pin, GPIO_PIN_SET);
```

Comment:
?

Reply:
The chip select pins are active low. Was still seeing some weird behavior on the logic analyzer until I set all the chip selects to be high.